ui/app/src/validation/project.ts

```diff
@@ -13,23 +13,31 @@
 
 import { z } from 'zod';
 import { useMemo } from 'react';
-import { resourceIdValidationSchema } from '@perses-dev/plugin-system';
 import { useProjectList } from '../model/project-client';
+import { generateMetadataName } from '../utils/metadata';
 
-export const projectNameValidationSchema = z.object({
-  name: resourceIdValidationSchema,
+export const projectDisplayNameValidationSchema = z
+  .string()
+  .min(1, 'Required')
+  .max(75, 'Must be 75 or fewer characters long');
+
+export const createProjectDialogValidationSchema = z.object({
+  projectName: projectDisplayNameValidationSchema,
```

Comment:
Why change name to projectName ? I like the fact that it's not project tainted as maybe one day we can have a common validation scheme for that WDYT?

Reply:
Because for dashboard validation schema, there is "projectName" and "dashboardName". Just wanted to align stuff 😄

Reply:
Again something not related then :x 

Is there inheritance in zod ? Like the good refactoring (don't do it there) woudn't be to say that? :
- the dashboard schema inherits from a common "with `name`" schema + add project validation
- the project schema inherits from the common "with `name`" schema

Reply:
You can extends / merge schema together, so yeah possible